interview-questions.md

```diff
@@ -8,44 +8,57 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: Super pulls in the values that you want the new class to inherit.
```

Comment:
All of your answers look great in this section, although I would strongly consider expanding on this first initial answer.